packages/mdc-fab/mdc-fab.scss

```diff
@@ -31,6 +34,26 @@
   @include mdc-fab--mini_;
 }
 
+.mdc-fab--extended {
+  @include mdc-typography(button);
+
+  width: auto;
+  padding: 0 20px 0 16px;
+  height: $mdc-fab-extended-height_;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  white-space: nowrap;
+  border-radius: $mdc-fab-extended-height_ / 2;
+}
+
+.mdc-fab__label {
+  padding-left: 8px;
+  justify-content: flex-start;
+  text-overflow: ellipsis;
```

Comment:
I don't think the ellipsis will ever show. I think you need a max-width or something.
<img width="1447" alt="screen shot 2018-06-01 at 10 22 06" src="https://user-images.githubusercontent.com/579873/40854272-ba4b6d08-6585-11e8-9244-95209a4ba856.png">

Reply:
This was required for fluid Extended FAB. But is is probably a good idea to add max-width to 100%.